Modules/pyexpat.c

```diff
@@ -1849,6 +1923,9 @@ MODULE_INITFUNC(void)
     MYCONST(XML_CQUANT_PLUS);
 #undef MYCONST
 
+    static struct PyExpat_CAPI capi;
```

Comment:
Why does this variable have to be static?

Reply:
@vstinner I was unsure about this actually.  None of the capsule methods use module state, and the address of capi is stored by PyCapsule_New.  If it is on the stack, it will be garbage memory later when it is used.

I can either put it in the module state (like I did in my latest changes to the unicodedata module) or make it static to keep it valid memory.  It was simpler to make it static and I thought it might be fine since none of the methods use module state.  i.e. no reason why importing the module and getting the capsule in multiple interpreters should give a different address.

Reply:
@vstinner what are next steps for this branch?

Reply:
I created https://bugs.python.org/issue41798 to track this specific variable. I can be enhanced later.